clients/iOS/Grapevine/Grapevine/Event.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Event.swift
+//  Grapevine
+//
+//  Created by Matthew Flickner on 9/29/15.
+//  Copyright © 2015 Grapevine. All rights reserved.
+//
+
+import UIKit
+import ObjectMapper
+
+class Event: NSObject, Mappable {
+    var title: String!
+    //var date: NSDate
+    //var status: String
+    //var id: Int
+    //var timeProcessed: NSDate
+    //var location: Location
+    //var startTime: NSDate
+    //var endTime: NSDate
+    //var repeatsWeekly: Bool
+    //var tags: [String]
```

Comment:
Just simplifying when I was setting up ObjectMapper. They'll be good to go in my next PR